cmd/gopogh/main.go

```diff
@@ -59,6 +60,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *outSummarySQLLitePath != "" {
+		if err := c.SQL(*outSummarySQLLitePath); err != nil {
+			fmt.Printf("%v", err)
```

Comment:
```suggestion
			fmt.Println(err)
```
This can be simplified